index.html

```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
             </div>
         </div>
     </section> 
+    <br>
+    <br>
+    <br>
+    <section class="tonic">
+        <div class="tonicimage">
+            <img src="img/Snapshoot Portfolio.png" alt="snapshot">
+        </div>
+        <div class="tonichead">
+            <p>Tonic</p>
```

Comment:
- [ ] Kindly ensure that all your cards' titles use header tag `hx` not `p` tag. 👍